src/s1reader/s1_reader.py

```diff
@@ -534,8 +559,8 @@ def _is_zip_annotation_xml(path: str, id_str: str) -> bool:
         return True
     return False
 
-def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str='vv',
-                burst_ids: list[str]=None):
+def load_bursts(path: str, orbit_path: str, swath_num: int, pol: str = 'vv',
```

Comment:
Are you sure this abide to pep8 convention? I thought that with parenthesis there should be no space around a `=` operator

Reply:
This was suggested by Codacy, so I assume it follows pep8.